### ISSUE TITLE
When I try to view a user's tweets, only my tweets are displayed.

### ISSUE THREAD

**Issue by kamina-zzz** [dimdenGD/OldTweetDeck#60]
When you open Profile modal as shown below and click `TWEETS` button at the bottom left, then you just get listed your tweets instead of user's tweets. But `MENTIONS` button works fine.
(画像の左下にある TWEETS ボタンをクリックしても自分の投稿しか表示されません。 MENTIONS ボタンは正しく動きます。)
![image](https://github.com/dimdenGD/OldTweetDeck/assets/5731602/acb1e089-b882-4fa2-a160-af42c612513e)

**Comment by PresumablyPaul** [comment]
That issue appears to happen when logged in on one account and setting a secondary account as the default in TweetDeck.

When the default account is set to the same account logged in, tweets within profiles display normally (as opposed to seeing tweets from your own account).
![sample](https://github.com/dimdenGD/OldTweetDeck/assets/143474048/088088a8-2d42-473d-aa90-8f39423084fa)

**Comment by kamina-zzz** [comment]
Oh, I see. I had forgotten about this setting. thx!

**Comment by PresumablyPaul** [comment]
You're welcome.

Previous to the need for OldTweetDeck, when viewing someone's profile, TweetDeck would successfully display a user's tweets regardless of which account was set as the default in TweetDeck. Now it will only do so properly when the default account matches the primary logged in account, otherwise it displays the tweets of the secondary account set as the default.

Previously, the request to view an account's tweets would be based on the account set as default in TweetDeck. So, for example, if "Bob" was set as default, and Bob attempted to view the tweets of an account that has blocked Bob, the tweets would not load/display. But if "Bob" set his secondary account as default, and that secondary account was not blocked, the user's tweets would display.

I think the issue right now might be related to or similar to the issue from this post: Ver3.0.0 [The user column only shows the main account. #38](https://github.com/dimdenGD/OldTweetDeck/issues/38)

Perhaps @dimdenGD will have a look into it...

**Comment by dimdenGD** [comment]
Fixed in 3.0.5

**Comment by PresumablyPaul** [comment]
Awesome! Damn, you're fast :)

Interestingly, I am testing 3.0.5 on MS Edge, and as you say the issue is fixed. But funnily enough, I am testing 3.0.2 on Vivaldi and it is now also displaying user tweets correctly, regardless of which of my accounts I set as the default. Perhaps there's some sort of background thing going on that I'm not aware of.

As a side note, it took awhile to get BTD to finally kick in on Edge (after many reloads), but it's quite consistent loading now. I'm the guy (same user name) who Tweeted to you about it yesterday. That said, I do find the scroll bar settings most often do not obey when changed.

I've noticed numerous people mention having problems with scroll bar settings in BTD. For testing purposes I am playing around with various combinations of Windows and internet browser scroll bar settings to see if perhaps there is some sort of conflict. Feel free to tell me to stop doing so if I am on the wrong train of thought. lol

Also still having trouble getting BTD to work with Vivaldi. It only works very rarely. *shrug*

Regardless, thanks so much for the effort you put in!
